packages/dui3/lib/bridge/sketchup.ts

```diff
@@ -51,7 +99,143 @@ export class SketchupBridge extends BaseBridge {
 
   // NOTE: Overriden emit as we do not need to parse the data back - the Sketchup bridge already parses it for us.
   emit(eventName: string, payload: string): void {
-    this.emitter.emit(eventName, payload as unknown as Record<string, unknown>)
+    const eventPayload = payload as unknown as Record<string, unknown>
```

Comment:
What's going on here - why is payload typed to be a string, and then suddenly expected to be an object?